### PR TITLE
Support Rsync to transfer clickhouse data file

### PR DESCRIPTION
**Pull request by Emor-nj**
## Purpose of this pull request
https://github.com/apache/incubator-seatunnel/issues/2073
Support Rsync to transfer clickhouse data file.

### CODE REVIEW

**Comment by Emor-nj** [comment]
@Hisoka-X Please have a look, and give me some advice.Thank you!

File: seatunnel-connectors/seatunnel-connectors-spark/seatunnel-connector-spark-clickhouse/pom.xml [hunk 1]
```diff
@@ -56,6 +56,12 @@
             <artifactId>sshd-scp</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.github.fracpete</groupId>
+            <artifactId>rsync4j-all</artifactId>
+            <version>3.2.3-10</version>
+        </dependency>
```
Comment (CalvinKirs) [inline seatunnel-connectors/seatunnel-connectors-spark/seatunnel-connector-spark-clickhouse/pom.xml]: https://mvnrepository.com/artifact/com.github.fracpete/rsync4j-all

The Apache License and GPL are not compatible and we cannot use this dependency

Comment (Emor-nj) [reply seatunnel-connectors/seatunnel-connectors-spark/seatunnel-connector-spark-clickhouse/pom.xml]: Thank you. I will try to use the buildin package scala.sys.process to build rsync shell command to realize this feature. And recommit it.

Comment (Hisoka-X) [reply seatunnel-connectors/seatunnel-connectors-spark/seatunnel-connector-spark-clickhouse/pom.xml]: > Thank you. I will try to use the buildin package scala.sys.process to build rsync shell command to realize this feature. And recommit it.

After you change, ping me. thx

Comment (Emor-nj) [reply seatunnel-connectors/seatunnel-connectors-spark/seatunnel-connector-spark-clickhouse/pom.xml]: @Hisoka-X i have changed and passed the simple test in spark-local mode , please check it.

**Review by Hisoka-X — DISMISSED** [review]
LGTM, well done!

**Comment by Hisoka-X** [comment]
Can you submit this feature to our new seatunnel api connector? If so, it's very wonderful.

**Comment by Emor-nj** [comment]
> Can you submit this feature to our new seatunnel api connector? If so, it's very wonderful.

Sure, I'm happy to do it.

**Review by Hisoka-X — APPROVED** [review]

**Pull request merged**